supportedBackends/local/src/main/scala/cromwell/backend/impl/local/LocalInitializationActor.scala

```diff
@@ -1,43 +1,59 @@
 package cromwell.backend.impl.local
 
+import java.nio.file.FileSystems
+
 import akka.actor.Props
 import better.files._
 import cromwell.backend.impl.local.LocalInitializationActor._
 import cromwell.backend.io.WorkflowPaths
 import cromwell.backend.validation.RuntimeAttributesKeys._
 import cromwell.backend.{BackendConfigurationDescriptor, BackendInitializationData, BackendWorkflowDescriptor, BackendWorkflowInitializationActor}
+import cromwell.filesystems.gcs.{GcsFileSystemProvider, GcsFileSystem}
 import wdl4s.types.{WdlBooleanType, WdlStringType}
 import wdl4s.{Call, WdlExpression}
+import LocalImplicits._
 
 import scala.concurrent.Future
 
 object LocalInitializationActor {
   val SupportedKeys = Set(DockerKey, FailOnStderrKey, ContinueOnReturnCodeKey)
 
-  def props(workflowDescriptor: BackendWorkflowDescriptor, calls: Seq[Call], configurationDescriptor: BackendConfigurationDescriptor): Props =
-    Props(new LocalInitializationActor(workflowDescriptor, calls, configurationDescriptor))
+  def props(workflowDescriptor: BackendWorkflowDescriptor, calls: Seq[Call], configurationDescriptor: BackendConfigurationDescriptor, localConfiguration: LocalConfiguration): Props =
+    Props(new LocalInitializationActor(workflowDescriptor, calls, configurationDescriptor, localConfiguration))
 
 }
 
 class LocalInitializationActor(override val workflowDescriptor: BackendWorkflowDescriptor,
                                override val calls: Seq[Call],
-                               override val configurationDescriptor: BackendConfigurationDescriptor) extends BackendWorkflowInitializationActor {
+                               override val configurationDescriptor: BackendConfigurationDescriptor,
+                               localConfiguration: LocalConfiguration) extends BackendWorkflowInitializationActor {
 
   override protected def runtimeAttributeValidators: Map[String, (Option[WdlExpression]) => Boolean] = Map(
     DockerKey -> wdlTypePredicate(valueRequired = false, WdlStringType.isCoerceableFrom),
     FailOnStderrKey -> wdlTypePredicate(valueRequired = false, WdlBooleanType.isCoerceableFrom),
     ContinueOnReturnCodeKey -> continueOnReturnCodePredicate(valueRequired = false)
   )
 
-  private val workflowPaths = new WorkflowPaths(workflowDescriptor, configurationDescriptor.backendConfig, None)
+  private val fileSystems = {
+    val maybeGcs = {
+      localConfiguration.gcsAuthMode map { authMode =>
+        val storage =authMode.buildStorage(workflowDescriptor.workflowOptions.toGoogleAuthOptions, localConfiguration.googleConfig)
```

Comment:
some spacing issues w/ the `=`

Also I'd love to not have 8 line val assignments but feel free to ignore me on that one